packages/core/src/RenderingEngine/WSIViewport.ts

```diff
@@ -428,6 +453,48 @@ class WSIViewport extends Viewport {
     this.refreshRenderValues();
   };
 
+  /**
+   * Converts a slide coordinate to a image coordinate using WSI utils functions
+   * @param point
+   * @returns
+   */
+  public worldToIndexWSI(point: Point3): Point2 {
+    const affine = this.viewer[Symbol.for('affine')];
+    const pixelCoords = WSIUtilFunctions?.applyInverseTransform({
```

Comment:
You don't need the ? here as it will throw anyways if not defined.  You could add a shortcut return if it isn't defined.
Applies to all the spots you use it.